cmd/minishift/cmd/start.go

```diff
@@ -612,6 +623,9 @@ func initStartFlags() *flag.FlagSet {
 	if minishiftConfig.EnableExperimental {
 		startFlagSet.Bool(configCmd.NoProvision.Name, false, "Do not provision the VM with OpenShift (experimental)")
 		startFlagSet.String(configCmd.ISOUrl.Name, minishiftConstants.B2dIsoAlias, "Location of the minishift ISO. Can be an URL, file URI or one of the following short names: [b2d centos minikube].")
+		startFlagSet.String(configCmd.RemoteIPAddress.Name, "", "IP address of the remote machine to run Minishift")
```

Comment:
suggestion `Specify IP address of the host to provision OpenShift`

Reply:
`IP address of the host to provision OpenShift on`

no need to use specify, as that is what an option is used for in general.